packages/docs/src/components/on-this-page/on-this-page.tsx

```diff
@@ -100,7 +100,7 @@ export const OnThisPage = component$(() => {
       <h6>More</h6>
       <ul>
         <li>
-          <a href={editUrl} target="_blank">
+          <a href={editUrl} rel="noopener" target="_blank">
```

Comment:
I saw this while collecting information so I searched for information and it would be good to keep this in mind for those browsers 👀. https://developer.chrome.com/docs/lighthouse/best-practices/external-anchors-use-rel-noopener/

![image](https://github.com/BuilderIO/qwik/assets/7875216/bb7ca9a3-5fa4-4dc4-a0c0-2e43a17aa4ba)